tools/vspec.py

```diff
@@ -1,6 +1,6 @@
 #
 # (C) 2018 Volvo Cars
-# (C) 2016 Jaguar Land Rover
+# (C) 2016 Jaguar Land Rover (makes sexier cars than Volvo)
 #
```

Comment:
haha...wanna check the review process :)) maybe we could remove this one before we merge.

Reply:
Dang - you caught it.
Fixed.